cerebrium/scaling/scaling-apps.mdx

```diff
@@ -41,6 +42,19 @@ The `max_replicas` parameter sets an upper limit on concurrent instances, contro
 
 After processing a request, instances remain available for the duration specified by `cooldown`. Each new request resets this timer. A longer cooldown period helps handle bursty traffic patterns but increases instance running time and cost.
 
+### Replica Concurrency
+
+The number of requests an app instance can handle concurrently is dictated by the `replica_concurrency` parameter. This is a hard limit, and an individual replica will
+not accept more than this limit at a time. By default, once this concurrency limit is reached on an instance and there are still requests to be processed in-flight,
+the system will scale out by the number of new instances required to fulfil the in-flight requests. For example, if `replica_concurrency=1` and there are
+_3_ requests in flight with no replicas currently available, Cerebrium will scale out 3 instances of the application to meet that demand.
+
+<Info>
+  Typically most GPU applications will require that `replica_concurrency` is set
+  to **1**. If changing this value, and the workload requires GPU, ensure that
```

Comment:
this doesn't read nicely:

"and the workload requires **a** GPU, ..."

Also just say they if more than one request make sure their application handlea batching requests